jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/utils/SqlNodesCreator.java

```diff
@@ -0,0 +1,76 @@
+package utils;
+
+import com.exasol.adapter.metadata.*;
+import com.exasol.adapter.sql.*;
+
+import java.math.*;
+import java.util.*;
+
+import static com.exasol.adapter.sql.AggregateFunction.AVG;
+
+/**
+ * This class contains static methods for fast creation of SQL nodes which are used in tests for SQL generation
+ * visitors. Helps to avoid duplication and speed up testing.
+ */
+public class SqlNodesCreator {
+    private SqlNodesCreator() {
+    }
+
+    public static SqlOrderBy createSqlOrderByDescNullsFirst() {
+        final List<SqlNode> orderByArguments = new ArrayList<>();
+        final ColumnMetadata columnMetadata = ColumnMetadata.builder().name("test_column").type(DataType.createBool())
```

Comment:
The problem with hidden constants is that this makes the test hard to read. You have a value that you compare against, but nowhere in the preparation of the test do you actually see that value.

Reply:
fixed